nodestream/cli/operations/generate_migration.py

```diff
@@ -0,0 +1,63 @@
+from ...schema import Schema
+from ...schema.migrations import Migration, MigratorInput, ProjectMigrations
+from .operation import NodestreamCommand, Operation
+
+
+class CleoMigrationInput(MigratorInput):
+    def __init__(self, command: NodestreamCommand):
+        self.command = command
+
+    def ask_yes_no(self, question: str) -> bool:
+        return self.command.confirm(question, default=False)
+
+
+class GenerateMigration(Operation):
+    def __init__(
+        self,
+        migrations: ProjectMigrations,
+        current_state: Schema,
+        dry_run: bool = False,
+    ):
+        self.current_state = current_state
+        self.migrations = migrations
+        self.dry_run = dry_run
+
+    def describe_migration(self, command: NodestreamCommand, migration: Migration):
+        num_changes = len(migration.operations)
+        command.line(f"Generated migration {migration.name}.")
```

Comment:
I wonder in general how helpful would some logging and error handling be in these cli commands. Or does the stacktrace pop up and it is easy to debug from there?

Reply:
See my message [here](https://github.com/nodestream-proj/nodestream/pull/237/files#r1480374760) and let me know what you think.